resources/css/index.css

```diff
@@ -0,0 +1,129 @@
+.header {
+  position: fixed;
+  z-index: 2;
+  top: 0;
+  left: 0;
+  right: 0;
+}
+
+.header-top {
+  padding: 20px 10px 15px;
+  background: #f72210;
+}
+
+.header-date {
+  color: white;
+  font-size: 16px;
+  font-weight: 400;
+  font-family: "Roboto Condensed", sans-serif;
+  line-height: 34px;
+  padding-left: 20px;
+}
+
+.header-logo {
+  cursor: pointer;
+}
+
+.header-logo svg {
+  fill: white;
+}
+
+.header-login-subscribe {
+  padding-right: 15px;
+}
+
+.header-links {
+  font-size: 19px;
+  font-weight: 700;
+}
+
+.categories-first {
+  border-left-width: 0;
+}
+
+.header-links-categories ul {
+  margin-bottom: 0;
+}
+
+.header-links-categories ul li a {
+  color: black;
+  text-decoration: none;
+}
```

Comment:
Avoid using custom styling where bootstrap styling is possible. Example in the above, you can text-decoration-none and text-dark bootstrap classes. Please check the rest of your styling as well.